samples/cellular/modem_trace_flash/README.rst

```diff
@@ -83,7 +83,6 @@ Testing
 After programming the sample and board controller firmware (as mentioned in Requirements section) to your development kit, test it by performing the following steps:
 
 #. |connect_kit|
-#. |connect_terminal|
```

Comment:
Then we should open the nRF Connect Serial terminal instead to observe the output. This can be opened separately or through the Cellular Monitor.

Reply:
That step is mentioned in the following steps. If it's a hard requirement I could remove the `(optional)` string.